solver/src/main/java/org/chocosolver/solver/variables/impl/AbstractVariable.java

```diff
@@ -73,6 +73,12 @@ public abstract class AbstractVariable implements Variable {
      */
     static final String MSG_BOUND = "new bounds are incorrect";
 
+    /**
+     * Empty bipartite list
+     */
+    static final ABipartiteList EMPTY = new ABipartiteList() {
```

Comment:
in the end the JVM will create an anonymous class, maybe we could have a proper class for the default case